README.md

```diff
@@ -39,19 +39,14 @@ With the ESnext field declarations proposal, the above example can be written as
 class Counter extends HTMLElement {
```

Comment:
@karlhorky there's potential performance differences, but without detailed benchmarking that's a tenuous thing to claim - but since there'd be N functions, instead of 1 function with N bind-passthroughs to it, it seems like the engine would be more able to optimize in the latter case.